WordPressKit/ReaderPostServiceRemote.m

```diff
@@ -200,7 +200,7 @@ - (void)fetchPostsFromEndpoint:(NSURL *)endpoint
                            success:(void (^)(NSArray<RemoteReaderPost *> *posts, NSString *algorithm))success
                            failure:(void (^)(NSError *))failure
 {
-    NSString *path = [endpoint absoluteString];
+    NSString *path = [endpoint path];
```

Comment:
I wonder how many of these migration issues are left in the library, and what it would take to discover them on our side before they reach the users... 🤔 

I suppose one way would be to audit each `GET...` call. There are "only" 118 of them...

Reply:
I haven't looked where the app gets `endpoint`. If they are constructed by the app, I'm thinking maybe we can dumb down this `fetchPostsFromEndpoint` function to not accepting a `URL` argument?